components/tidb_query_expr/src/impl_time.rs

```diff
@@ -150,6 +150,50 @@ pub fn to_seconds(ctx: &mut EvalContext, t: &DateTime) -> Result<Option<Int>> {
     Ok(Some(t.second_number()))
 }
 
+#[rpn_fn(nullable, capture = [ctx])]
```

Comment:
Please considering remove `nullable` here so we don't need to check this, take [this PR](https://github.com/tikv/tikv/pull/8728) as ref.

Reply:
ditto

Reply:
done

PTAL

Reply:
done

PTAL

Reply:
Seems that `nullable` is not removed. Do you forget to push latest code?